plugins/view-resources/src/components/Move.svelte

```diff
@@ -106,7 +106,9 @@
   <div class="spaceSelect">
     {#await getSpace() then}
       {#if currentSpace}
-        <SpaceSelect _class={currentSpace._class} label={_class ?? ''} bind:value={object.space} />
+        {#if _class}
```

Comment:
Can we write `{#if currentSpace && _class}` to get rid of nested `if`s?

Reply:
Thanks, fixed.